base/tests/test_GridFactory.cpp

```diff
@@ -411,11 +411,10 @@ BOOST_AUTO_TEST_CASE(testOperationMultipleEval) {
   GridGenerator& gen = factory->getGenerator();
   gen.regular(2);
 
-  DataVector alpha(factory->getSize());
+  DataVector alpha(factory->getSize(), 0.0);
```

Comment:
Is this necessary? I thought DataVector initializes to 0 by default. Also check other occurrences below.

Reply:
The call is not technically necessary from a strictly programming perspective, but it's more a design choice by me, to emphasize that you can, and probably makes sense, to put there whatever value you want. Other tests in this test file use other initial values for alpha, so it was more for consistency in this case... Anyway, it is the same number of operations, as the default constructor calls this one, with value 0.

Reply:
I see. That makes sense.